products/accessapproval/terraform.yaml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  FolderSettings: !ruby/object:Overrides::Terraform::ResourceOverride
+    legacy_name: "google_folder_access_approval_settings"
+    import_format: ["folders/{{folder_id}}/accessApprovalSettings"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        skip_test: true
```

Comment:
just curious: Does the generated test end up not working? `org_id` should (theoretically) cause a working test to be created.

Reply:
No, the generated test works, however, I wanted to test update, and I was afraid if this test ran in parallel to the handwritten tests, the results may conflict due to the fact there is only one settings resource to update, so I'm just skipping this one and the same thing will be tested in the handwritten test, only with testing update as well.